docs/releases.md

```diff
@@ -0,0 +1,65 @@
+# Release process
+
+The release process in broken up into 3 steps.
+
+1. Increment the version and add a new tag
+2. Modify the changelog
+3. Merge the release branch and create a "github release"
+
+## 1. Increment the version and add a new tag
+
+Run
```

Comment:
Maybe replace "Run" with what the command will do. Something like "To update the version in package.json and create a tag run:".